.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
looks like this file is lacking the `insert_final_newline`? 😀

Reply:
Yes, python files usually have 4 spaces. I have fixed it and add the final newline style for the .md file. Thanks.